app/assets/styles/emergency/_global.scss

```diff
@@ -0,0 +1,131 @@
+.page--emergency {
+	.fold {
+    margin-bottom: $global-spacing * 2;
+  }
+}
+
+.introduction-nav {
+  list-style: none;
+  margin-left: 0;
+  li {
+    display: inline-block;
+    margin-right: $global-spacing;
+    text-transform: uppercase;
+    font-size: 0.75rem;
+  }
+}
+
+.introduction-nav-item--active {
+  font-weight: 600;
+}
+
+.inpage__nav {
+  border-top: solid 1px rgba($base-color, 0.1);
+  font-family: $heading-font-family;
+  padding: ($global-spacing * 2) 0 $global-spacing 0;
+  > .inner {
+  @extend .row, .row--centered;
+  }
+
+  ul {
+    list-style: none;
+    margin-left: 0;
+    display: flex;
+    align-items: stretch;
+    justify-content: space-between;
+    width: 100%;
+  }
+
+  li {
+    display: inline-block;
+    flex: 0 1 auto;
+    margin-right: $global-spacing * 4;
+    &:last-child {
+      margin-right: 0;
+    };
+  }
+}
+
+.key-figures-list {
+  list-style: none;
+  @include column(12/12);
+  margin-left: 0;
+  display: block;
+  @include media(medium-up) {
+    display: flex;
+    align-items: stretch;
+    ustify-content: space-between;
+  }
+
+  li {
+    text-align: center;
+    display: inline-block;
+    border: solid 1px rgba($base-color, 0.1);
+    border-radius: $base-border-radius;
+    padding: ($global-spacing * 2) ($global-spacing * 1);
+    width: 100%;
+    margin-bottom: $global-spacing;
+    @include media(medium-up) {
+      padding: ($global-spacing * 4) ($global-spacing * 2);
+      margin-left: $global-spacing * 2;
+      margin-bottom: 0;
+      &:first-child {
+        margin-left: 0;
+      };
+    }
+  }
+
+  h3 {
+    color: $primary-color;
+    font-family: $base-font-family;
+    font-size: 2.5rem;
+    font-weight: 300;
+    margin-bottom: $global-spacing/2;
+  }
+}
+
+.key-figure-label {
+  font-weight: 600;
+  font-size: 1rem;
+  margin-bottom: 0;
+}
+
+.key-figure-source { 
+  color: rgba($base-color, 0.48);
+  margin-bottom: 0;
+}
+
+.situation-reports-list {
+  list-style: none;
+  margin-left: 0;
+  li {
+    @include media(medium-up) {
+      @include column(3/10);
+    }
+    @include column(12/12);
```

Comment:
It would be clearer if the element's properties came before the media queries definition